src/components/Bookings/Bookings.jsx

```diff
@@ -5,14 +5,21 @@ import AddBookingForm from "../AddBookingForm/AddBookingForm";
 
 const Bookings = () => {
   const [bookings, setBookings] = useState([]);
+  const [loading, setLoading] = useState(true); // Define loading state
 
   useEffect(() => {
     console.log("Component is mounted");
     fetch("https://nw6-cyf-hotel.glitch.me/fakebookings")
       .then(response => response.json())
-      .then(data => setBookings(data))
-      .catch(error => console.error("Error fetching bookings:", error));
-  }, []);
+      .then(data => {
+        setBookings(data);
+        setLoading(false);
+      })
+      .catch(error => {
+        console.error("Error fetching bookings:", error);
```

Comment:
Don't forget to display errors to users not to the console - you look in the console but the average user won't :)